### PR TITLE
Release v12.9

### PR DESCRIPTION
**Pull request by polmih**
Release v12.9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "GCWeb",
-  "version": "12.8.1",
+  "version": "12.9",
```
Comment (duboisp) [inline package.json:3]:
```suggestion
  "version": "12.9.0",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "GCWeb",
-  "version": "12.8.1",
+  "version": "12.9",
```
Comment (duboisp) [inline package-lock.json:3]:
```suggestion
  "version": "12.9.0",
```

**Review by duboisp — CHANGES_REQUESTED** [review]
It need the 3 digit for the versioning number

File: package.json (hunk 1 above)
Comment (polmih) [reply package.json]: Fixed. Thanks

File: package-lock.json (hunk 2 above)
Comment (polmih) [reply package-lock.json]: Fixed. Thanks

File: index-en.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ css:
 			<p>The page templates and design patterns below comprise a reference implementation of the <a href="https://design.canada.ca">Canada.ca design system</a>, including the mandatory requirement of the Content and Information Architecture (C&amp;IA) Specification. Government of Canada departments and agencies can contribute additional patterns and templates via <a href="https://github.com/wet-boew/GCWeb">GCWeb github repository</a>.</p>
 		</div>
 		<div class="col-xs-12 col-md-auto pull-right">
-			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.8.1.zip" class="btn btn-primary">Download GCWeb theme <strong>v12.8.1</strong></a><br />
+			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.9.zip" class="btn btn-primary">Download GCWeb theme <strong>v12.9</strong></a><br />
```
Comment (duboisp) [inline index-en.md:21]:
```suggestion
			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.9.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v12.9.0</strong></a><br />
```

File: index-fr.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ css:
 			<p>Les gabarits et les conceptions communes si dessous sont une référence d'implémentation du <a href="https://conception.canada.ca">Système de conception de Canada.ca</a>, incluant les exigences obligatoire de la spécifications du contenu et de l’architecture de l'information (C&amp;AI) pour Canada.ca. Les ministères et organisme du gouvernement du Canada peuvent y contribuer en publiant leur modèle et leur conception commune via le <a href="https://github.com/wet-boew/GCWeb">dépôt github de GCWeb</a>.</p>
 		</div>
 		<div class="col-xs-12 col-md-auto pull-right">
-			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.8.1.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v12.8.1</strong></a><br />
+			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.9.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v12.9</strong></a><br />
```
Comment (duboisp) [inline index-fr.md:21]:
```suggestion
			<p><a href="https://github.com/wet-boew/GCWeb/archive/v12.9.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v12.9.0</strong></a><br />
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Links to update

**Review by duboisp — APPROVED** [review]

**Pull request merged**